some.c

```diff
@@ -0,0 +1,3 @@
+system(string);
```

Comment:
:exclamation:**Cycode**: SAST violation: 'The software constructs an OS command using externally-influenced input,  but it does not neutralize or incorrectly neutralizes special elements  that could modify the intended OS command.'.
 
Severity: High


Would you like to exclude this SAST violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_sast_ignore_here** | Applies to this request only |
| **#cycode_sast_false_positive** | Applies to this SAST violation for all repos in your organization |

 <!-- 5c9ab3aa-54b7-4eaf-8c0e-357993c91b3b -->